setup.py

```diff
@@ -0,0 +1,83 @@
+# Copyright 2017 Mycroft AI Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+from setuptools import setup, find_packages
+import os.path
+
+BASEDIR = os.path.abspath(os.path.dirname(__file__))
+
+
+def get_version():
```

Comment:
Since these functions used to be in `mycroft.util.setup_base`, are we able to remove everything in that file now.

Reply:
I'll remove the references to it and remove it then. I was a bit hesitant since it was part of the distribution

Reply:
I've only seen it used twice in `mycroft/metrics/__init__.py` which can be replaced by `CORE_VERSION_STR`.